### DISCUSSION TITLE
About Themes and Layout when developping a new theme

### DISCUSSION THREAD

**Discussion by happytodev** [Cecilapp/Cecil#1885] [Q&A] [answered]
Hello @ArnaudLigny ,

I'm wondering when I develop a theme how can I say that, for example, the `themes/themename/layouts/blog/custom.html.twig` must replace the default one in `layouts/blog/page.html.twig`.

Currently, the only way I found is to set in my blog post in Markdown 
`layout: custom`
to be sure Cecil use the layout of this theme.

If I don't do this, the `page` layout for blog folder is used instead.

I'm pretty sure there is another way more smart to do this.

Could you help me, please ?

See you.

**Comment by ArnaudLigny** [comment, marked as answer]
Hello @happytodev, you must see a theme as "default" layouts (and assets) used to render pages. So, you don't need local layouts, unless you want to override a theme layouts.

In your example you should have:

- `themes/<theme>/layouts/blog/page.html.twig` is used to render blog posts
- `layouts/blog/page.html.twig` if you want to replace the theme blog/post layout, otherwise it's not needed

**Reply by happytodev** [reply]
Thanks Arnaud for your answer.

To be perfectly clear, does that mean I can remove the default provided folder /layouts at the root of my project if my theme covers all the layouts I need?

**Reply by ArnaudLigny** [reply]
Yes, exactly.
